appserver/extras/docker-images/basic/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM @docker.java.image@
+
+ENV HOME_DIR=/opt/payara
+ENV PAYARA_DIR=${HOME_DIR}/appserver \
+    SCRIPT_DIR=${HOME_DIR}/scripts \
+    CONFIG_DIR=${HOME_DIR}/config \
+    DEPLOY_DIR=${HOME_DIR}/deployments \
+    PASSWORD_FILE=${HOME_DIR}/passwordFile \
+    ADMIN_USER=admin \
+    ADMIN_PASSWORD=admin \
+    JVM_ARGS="" \
+    MEM_MAX_RAM_PERCENTAGE="70.0" \
+    MEM_XSS="512k"
+ENV PATH="${PATH}:${PAYARA_DIR}/bin"
+
+ARG TINI_VERSION=v0.18.0
```

Comment:
Heads up that tini v0.19 has been released two days ago.
Releases now offer SHA256 sums, thus easing installation A LOT!
https://github.com/krallin/tini/releases/tag/v0.19.0

Reply:
I think we will merge this first and then upgrade this in standalone PR, but thanks for the good news!